process_report/process_report.py

```diff
@@ -72,6 +97,12 @@ def main():
         nargs="+",
         help="One or more CSV files that need to be processed",
```

Comment:
This will always require at least 1 arguments. But if we pass `--upload-to-s3` then we won't be passing in any `csv_files`. You need to update this to handle both cases. Don't know from the top of my head what argparse has in the bag for a scenario like this, maybe argparse subparser?